doc/pages/Manual.md

```diff
@@ -571,8 +571,8 @@ The following are allowed in filters:
 - Idents
 - Parentheses
 - Logical operators (binary AND `&`, binary OR `|`, prefix, unary NOT `!`)
-- The unary operator `?` for detecting whether an expression contains undefined
-  variables
+- The unary operator `?` for detecting whether an expression evaluates to a
+  defined value
```

Comment:
```suggestion
  defined value. If the expression `expr` is an undefined value then `?expr` will be evaluated to `false`. If it does not then it is evaluated to `true`.
```
We could also simplify and linking to the example 2 paragraphs below

Reply:
putting it in full sounds good to me, but I think this should mention evaluation: `?expr` is valid for all types `expr` and it will actually evaluate `expr`. How about something like:

> - The unary operator `?` for detecting whether a value is defined. `?expr` evaluates to `false` if `expr` evaluates to an undefined value, and `true` otherwise.

It's in a weird order, but this avoids talking about "defined values"

> - The unary operator `?` for detecting whether a value is defined. `?expr` evaluates to `true` if `expr` evaluates to a defined value, and `false` if it evaluates to an undefined value.

Reply:
I like the second proposal ^_^

Reply:
I updated to that